app/views/placements/show.html.erb

```diff
@@ -1,7 +1,9 @@
-<h3>Placement details</h3>
-<br>
+<div class='mt-4 mb-4'>
+    <h3>Placement details</h3>
+</div>
+
 
-<div class='card bg-light col-4 rounded mb-2'>
+<div class=' shadow-sm card bg-light col-4 rounded mb-2'>
```

Comment:
[Style] 
Please remove the extra space here.

```suggestion
<div class='shadow-sm card bg-light col-4 rounded mb-2'>
```

Reply:
Done